.github/workflows/tests.yml

```diff
@@ -5,45 +5,47 @@ on:
   schedule:
     - cron: '* * * * 0'  # run once a week (sunday)
```

Comment:
I've seen some GH workflows that also have `pull-request:` inside `on:`.  Is that not necessary to run checks on pull requests?

Does the answer change if the pull request is made from a fork rather than a branch in the same repo?

Reply:
On another note, if this cron specification follows `crontab` syntax then you're saying "run this every minute of every hour on Sunday". You're missing a minute (first element) and hour (second element) specification, right?

Reply:
@mgedmin `push:` seems to be enough if the PR is on a branch in the repo. I'll add `pull-request:` so hopefully PRs of forks are run, too.

Reply:
@dataflake Thank you for catching this, I am going to change it to `0 12 * * 0`.

Reply:
Does GHA support the `@daily`/`@weekly`/`@monthly` syntax?

*checks*

[No](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#scheduled-events) :disappointed: